frontend/amundsen_application/static/js/features/CodeBlock/index.tsx

```diff
@@ -0,0 +1,32 @@
+// Copyright Contributors to the Amundsen project.
+// SPDX-License-Identifier: Apache-2.0
+
+import * as React from 'react';
+import { CopyBlock, atomOneLight } from 'react-code-blocks';
+import './styles.scss';
+
+const LANGUAGE = 'sql';
+
+type CodeBlockProps = {
+  text: string;
+  children?: JSX.Element;
+};
+
+const CodeBlock: React.FC<CodeBlockProps> = ({
+  text,
+  children,
+}: CodeBlockProps) => (
+  <div className="code-block-container">
+    {children}
+    <CopyBlock
```

Comment:
Moved this from the dashboard page but made the following changes:

- Moved and refactored styles to work outside of the QueryListItem containers
- Added 'children' prop so the buttons can be added inside the CodeBlock component
- Added a wrapper class `code-block-container`